content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -42,35 +42,71 @@ Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
-## Deploy all Kubernetes clusters to be used in the mesh
-
-After deployment of remote clusters, each one will have a
-credentials file associated with the admin context typically
-located in `$HOME/.kube/config`.
+## Create service account in remote clusters
+
+The Istio control plane requires access to all clusters in the mesh to
+discover services, endpoints, and pod attributes.  The following
+describes how to create a Kubernetes service account with the minimal
+RBAC access required.  The procedure then generates a `kubeconfig` file
+for the cluster using the credentials of the service account.
+
+Perform these steps for each remote cluster in the mesh:
+
+1. Create `ClusterRole`, `istio-reader`, for the Istio control plane access:
+
+     ```command
+      cat <<EOF | kubectl  create -f -
+      kind: ClusterRole
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: istio-reader
+      rules:
+         - apiGroups: ['']
+           resources: ['nodes', 'pods', 'services', 'endpoints']
+           verbs: ['get', 'watch', 'list']
+    EOF
+    ```
 
-## Gather credential files from remote
+1.  Create `ServiceAccount`, `istio-multi`, for the Istio control plane:
 
-> `${CLUSTER_NAME}` here is defined as the name of the remote
-cluster used and must be unique across the mesh.  Some Kubernetes
-installers do not set this value uniquely.  In this case, manual
-modification of the `${CLUSTER_NAME}` fields must be done.
+    ```command
+    $ export SERVICE_ACCOUNT=istio-multi
+    $ kubectl create sa ${SERVICE_ACCOUNT}
+    ```
 
-For each remote cluster, execute the following steps:
+1.  Bind `ServiceAccount`, `istio-multi`, to `ClusterRole` `istio-reader`:
 
-1. Determine a name for the remote cluster that is unique across
-all clusters in the mesh.  Substitute the chosen name for the
-remaining steps in `${CLUSTER_NAME}`.
+    ```command
+    $ kubectl create clusterrolebinding istio-multi --clusterrole=istio-reader --serviceaccount=default:${SERVICE_ACCOUNT}
```

Comment:
This service account is created for the "default" namespace. Is this intended? Or any namespace is ok?

Reply:
Since its a ClusterRoleBinding any namespace is ok.  But I guess it might be better to put it in an istio-system namespace.  Maybe that fits better with scenarios where the default namespace has "admin" users that might be able to read the service-accounts in the namespace.